openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml

```diff
@@ -56,6 +56,24 @@
         0.73825
       ]
 
+- name: test WH1 peak demand annual savings
+  period: 2024
+  absolute_error_margin: 0.1
+  input:
+    SYS2_peak_demand_savings_capacity:
```

Comment:
This isn't right. This file is for WH1 but the variable is SYS2.

Reply:
File has now been updated to reflect correct variable.